packages/lisk-transaction-pool/src/transaction_pool.ts

```diff
@@ -13,89 +13,159 @@
  *
  */
 import { Queue } from './queue';
+import * as queueCheckers from './queue_checkers';
 
-// tslint:disable
 export interface TransactionObject {
 	readonly id: string;
+	receivedAt?: Date;
 	readonly recipientId: string;
 	readonly senderPublicKey: string;
 	readonly type: number;
-	receivedAt?: Date;
 }
 
 export interface TransactionFunctions {
-	verifyTransactionAgainstOtherTransactions?(
-		transaction: Transaction,
+	containsUniqueData(): boolean;
+	verifyTransactionAgainstOtherTransactions(
```

Comment:
Leaving a comment here for future naming revision: `verifyTransactionAgainstOtherTransactions` -> `verifyAgainstOtherTransactions`